tests/ast-alignment/parse.ts

```diff
@@ -67,7 +67,7 @@ function parseWithTypeScriptESTree(
 interface ASTComparisonParseOptions {
   parser: string;
   typeScriptESTreeOptions?: ParserOptions;
```

Comment:
Arent' these the wrong ones? Coming from `import { ParserOptions } from '@babel/parser';`